spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -75,27 +75,28 @@ private void rewrite() {
   private void rewriteWhereStat(ListIterator<Statement> stats) throws InvalidSpecCompileException {
     Statement stat = stats.next();
     BinaryExpression binExpr = AstUtil.getExpression(stat, BinaryExpression.class);
-    if (binExpr == null || binExpr.getClass() != BinaryExpression.class) // don't allow subclasses like DeclarationExpression
-      notAParameterization(stat);
-
-    int type = binExpr.getOperation().getType();
-
-    if (type == Types.LEFT_SHIFT) {
-      Expression leftExpr = binExpr.getLeftExpression();
-      if (leftExpr instanceof VariableExpression)
-        rewriteSimpleParameterization(binExpr, stat);
-      else if (leftExpr instanceof ListExpression)
-        rewriteMultiParameterization(binExpr, stat);
-      else
+    if (binExpr != null && binExpr.getClass() == BinaryExpression.class) // don't allow subclasses like DeclarationExpression
+    {
```

Comment:
Spock normally uses Egyptian-style braces

Reply:
ah, good catch.
IJ didn't insert it at the right place because of the line comment.